### PR TITLE
Next project structure

### PR DESCRIPTION
**Pull request by EmilK15**
Needed to update the new folder structure created by create-webiny-project.
This included descriptions of the new files added to the project.

### CODE REVIEW

File: docs/deep-dive/project-structure.md [hunk 1]
```diff
@@ -8,54 +8,36 @@ In this article we'll look at the project structure and explain the responsibili
...
-│   ├── project-utils/
-│   └── theme/
-├── .env.json
-├── .gitignore
-├── .prettierrc.js
-├── README.md
-├── package.json
-└── webiny.js
-```
+![New Project Folder Structure](/img/deep-dive/project-structure/folder-structure.png)
```
Comment (Ashu96) [inline docs/deep-dive/project-structure.md:11]: I prefer markdown tree version for project structure like we currently have

What do you think @webiny/webinydevteam ?

Comment (SvenAlHamad) [reply docs/deep-dive/project-structure.md:11]: I think in some cases that might cause a lot of typing if we have lots of files to show unless we can find a plugin for vs code or something that can export that for us automatically.

Comment (Pavel910) [reply docs/deep-dive/project-structure.md:11]: you an do it easily using the `tree` command. Built-in on Linux, installable using `brew` on OSX, maybe even works on Windows in `git bash`. 

Simply run ` tree . -L 2` to create a tree structure:
```
.
├── api
│   ├── apolloGateway
│   ├── databaseProxy
│   ├── example.env.json
│   ├── files
│   ├── formBuilder
│   ├── headless
│   ├── i18n
│   ├── pageBuilder
│   ├── resources.js
│   ├── security
│   ├── services
│   └── webiny.config.js
├── apps
│   ├── admin
│   ├── databaseProxy
│   ├── future
│   ├── resources.js
│   └── site
├── example.env.json
└── webiny.root.js
```

Comment (Pavel910) [reply docs/deep-dive/project-structure.md:11]: @EmilK15 you're using WSL so you can easily use this.

Comment (EmilK15) [reply docs/deep-dive/project-structure.md]: Added the tree-view. Also updated a few things I noticed about `apps` and `webiny.config.js` locations for users to edit.

**Comment by EmilK15** [comment]
Is this all good to merge?

**Comment by Pavel910** [comment]
@EmilK15 let it sit here for a while, I'll need to update the text here.

**Pull request merged**